research/sentiment_analysis/sentiment_main.py

```diff
@@ -30,15 +38,15 @@ def run_model(dataset_name, emb_dim, voc_size, sen_len,
                 optimizer="rmsprop",
                 metrics=["accuracy"])
 
-  print("loading the data")
+  tf.logging.info("loading the data")
   x_train, y_train, x_test, y_test = dataset.load(
       dataset_name, voc_size, sen_len)
 
   model.fit(x_train, y_train, batch_size=batch_size,
             validation_split=0.4, epochs=epochs)
   score = model.evaluate(x_test, y_test, batch_size=batch_size)
-  print("score")
-  print(score)
+  tf.logging.info("score")
+  tf.logging.info(score)
```

Comment:
nit: Score: ### would be more clear and concise in the logs.

Reply:
ack

Reply:
Sorry, I meant-- tf.logging.info("Score: {}".format(score)) or similar.